custom/conf/app.example.ini

```diff
@@ -549,6 +549,12 @@ COOKIE_REMEMBER_NAME = gitea_incredible
 ; Reverse proxy authentication header name of user name
 REVERSE_PROXY_AUTHENTICATION_USER = X-WEBAUTH-USER
 REVERSE_PROXY_AUTHENTICATION_EMAIL = X-WEBAUTH-EMAIL
+; Interpret X-Forwarded-For header or the X-Real-IP header and set this as the remote IP for the request
+REVERSE_PROXY_LIMIT = 1
+; List of IP addresses seperated by comma of trusted proxy servers. Use `*` to trust all.
+REVERSE_PROXY_TRUSTED_PROXIES = 127.0.0.1
+; List of network addresses seperated by comma of trusted proxy servers. Example `10.0.0.0/24`.
+REVERSE_PROXY_TRUSTED_NETWORKS =
```

Comment:
I think you should combine these two under `REVERSE_PROXY_TRUSTED_NETWORKS`. Single IPs can be specified as `/32` (v4) `/128` (v6): Loopback is BTW defined as `127.0.0.1/8`, not a single IP which is a common misconception.

Reply:
We could probably use single `REVERSE_PROXY_TRUSTED_PROXIES` and if string contains `/` use it as network, if not as IP address. I can change default to `127.0.0.0/8,[::1]/128`

Reply:
@silverwind done

Reply:
Thanks, config looks good to me now.